### PR TITLE
move actionButton to Aphrodite

### LINKED ISSUE #9163: move actionButton to Aphrodite

### PR DESCRIPTION
**Pull request by cezaraugusto**
Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Auditors: @luixxiul

Close #9163

Test Plan:

1. go to about:certerror
2. buttons should have no visual regressions
3. go to `aihsdiuahsdis.cisdhjfsidf` or any other invalid url (which will lead you to about:error page)
4. buttons should have no visual regressions
5. go to about:styles
6. action button should have no visual regressions
7. go to brave.com, block scripts
8. click no script button
9. buttons should have no visual regressions

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

**Comment by luixxiul** [comment]
I'll review soon.

File: app/renderer/components/common/browserButton.js [hunk 1]
```diff
@@ -16,12 +16,14 @@ class BrowserButton extends ImmutableComponent {
       this.props.subtleItem && [styles.browserButton_default, styles.browserButton_subtleItem],
       this.props.extensionItem && styles.browserButton_extensionItem,
       this.props.groupedItem && styles.browserButton_groupedItem,
-      this.props.notificationItem && styles.browserButton_notificationItem
+      this.props.notificationItem && styles.browserButton_notificationItem,
+      // actionItem is just subtleItem with a blue background
+      this.props.actionItem &&
+        [styles.browserButton_default, styles.browserButton_subtleItem, styles.browserButton_actionItem]
```
Comment (luixxiul) [inline app/renderer/components/common/browserButton.js]: I would move the 3 lines above to a line between L16-L17, `subtleItem` and `extensionItem`.

**Comment by luixxiul** [comment]
> go to about:error

Change looks good, while opening about:error directly shows a blank tab without buttons.

**Comment by cezaraugusto** [comment]
bad str, you should be lead to the err page typing an invalid url like `aihsdiuahsdis.cisdhjfsidf`, I just updated STR.

File: app/renderer/components/common/browserButton.js (hunk 1 above)
Comment (cezaraugusto) [inline app/renderer/components/common/browserButton.js]: I agree

**Pull request merged**